fixtures/js-stubs/types.tsx

```diff
@@ -95,6 +95,7 @@ type TestStubFixtures = {
   PullRequest: OverridableStub;
   Release: (params?: any, healthParams?: any) => any;
   Repository: OverridableStub;
+  RepositoryProjectPathConfig: OverridableStub;
```

Comment:
you can add the types as the template parameter for `OverridableStub` as well, I just didn't when I translated these all over

Reply:
I can do it in a follow-up